recipe/meta.yaml

```
@@ -13,6 +13,8 @@ source:
   - url: https://github.com/PennyLaneAI/pennylane-lightning/archive/refs/tags/v{{ version }}.tar.gz
     sha256: da9912f0286d1a54051cc19cf8bdbdcd732795636274c95f376db72a88e52d85
     folder: pennylane-lightning
+    #patches:
+    #  - enable_blas.diff
 
 build:
   number: 1
```

I noticed the hash changed (likely due to the change in the build/host requirements), we should bump this before merging.
```suggestion
  number: 2
```